examples/pytorch/text-classification/run_glue.py

```diff
@@ -503,23 +513,32 @@ def take_train_steps(model, trainer, resume_from_checkpoint, last_checkpoint):
     def train_func(model):
         return take_train_steps(model, trainer, resume_from_checkpoint, last_checkpoint)
 
+    if model_args.benchmark:
+        from optimum.intel.neural_compressor import IncOptimizedConfig
+        if model_args.int8:
+            print("runing int8 mode...")
+            optimized_config = IncOptimizedConfig.from_pretrained(training_args.output_dir)
+            from optimum.intel.neural_compressor.utils import load_inc_config
+            model = load_inc_config(model, optimized_config.path)
+
+        metric = take_eval_steps(model, trainer, metric_name)
```

Comment:
What do think about replacing this part with a loading verification at the end of the script ? (such as https://github.com/huggingface/optimum/blob/add-lpot-loading-quant-model/examples/pytorch/text-classification/run_glue.py#L582 ) Otherwise the behaviour is not very clear as the user should provide a directory with a quantized model / optimized config.

Reply:
@echarlaix Thanks for your quick comments. The idea here is to introduce a simply simulated deployment environment to run the inference/benchmark. So we assumed people will have a directory with a quantized model/optimized config. Your proposed change is also making a lot of sense, especially for the result checker. Some general questions: does the example under Optimum focus more on the functionality of model quantization? Where do we plan to verify the performance or end-to-end flow? I am thinking whether we may bring some end-to-end examples in transformer repo (link to Optimum) to give people overall idea?